src/main/java/classes/Bulldozer.java

```diff
@@ -0,0 +1,39 @@
+package classes;
+
+public class Bulldozer extends Machine {
+    private int operatingWeight;
+
+    public Bulldozer() {
+    }
+
+    public Bulldozer(String brand) {
+        super(brand);
+    }
+
+    public Bulldozer(int motorPower) {
+        super(motorPower);
+    }
+
+    public Bulldozer(String brand, int motorPower, int operatingWeight) {
+        super(brand, motorPower);
+        this.operatingWeight = operatingWeight;
+    }
+
+    public int getOperatingWeight() {
+        return operatingWeight;
+    }
+
+    public void setOperatingWeight(int operatingWeight) {
+        this.operatingWeight = operatingWeight;
+    }

```

Comment:
Reread the condition of the problem. All you need are two methods (no fields) and in these methods you have to write (machine name) start/stop work

Reply:
I removed unnecessary staff.  Had an issue with maven: "org.apache.maven.doxia.siterenderer.documentcontent".  Resolved by adding 2 plugins to pom.xml:

![image](https://user-images.githubusercontent.com/105455921/197361525-3da13299-23fe-4b5c-896f-09f98740f975.png)